modules/chat/public/views/partials/message_button.liquid

```diff
@@ -0,0 +1,14 @@
+{% liquid
+	if conversation.models.results[0].id
+		assign params = '?active_conversation=' | append: conversation.models.results[0].id
+	else
+		assign params = '?active_conversation=new&from_id=' | append: current_profile.id | append: '&to_id=' | append: user.id
```

Comment:
@z-x I believe this can be made with filter https://documentation.platformos.com/api-reference/liquid/platformos-filters#expand_url_template 
It uses https://www.rfc-editor.org/rfc/rfc6570.txt

Reply:
I would think that it's easier to understand with append, but I tried and I don't understand why this throws an error of 
`Expected end_of_string but found id in "{{template | expand_url_template conversation_id: 'test'}}`

`
	assign template = '?active_conversation={conversation_id}&from_id={from_id}&to_id={to_id}'

	if conversation.models.results[0].id
		assign params = template | expand_url_template conversation_id: conversation.model.results[0].id
	else
		assign params = template | expand_url_template conversation_id: 'new', from_id: current_profile.id, to_id: user.id
	endif
`